### ISSUE TITLE
way OF doing something

### ISSUE THREAD

**Issue by esmanning** [carmls/snacs-guidelines#134]
> He has no way OF guessing the danger
lpp_24
sent_id = lpp_1943.1228

We encountered this and were unsure about it. Checking for this construction in Reddit data, it looks like we haven't been consistent about it:

> For example, one of the sassier members of the party referred to the sorceress as a spoiled princess as a way OF insulting her
reddit task_013
new_doc id = 0cbdc014-dbed-b42e-6da3-13e783e9973b
sent_id = 9

 This is currently labeled Species, but eh

> as more and more of them get caught they'll learn not to steal cars and opt for other crimes or maybe even for a non-criminal way OF existence
reddit task_001
new_doc id = 022ee17b-a59d-43d6-65c9-90b2acb26b87
sent_id = 6

This is labeled Gestalt

**Comment by nschneid** [comment]
Is this the same as "a new way OF thinking" (Gestalt in the guidelines)?

**Comment by esmanning** [comment]
Oops, missed that!

Not sure if these are all Gestalt, or if there are two different things:
* new way OF thinking, non-criminal way OF existence - Gestalt
* no way OF guessing the danger, as a way OF insulting her - I think these could be more easily paraphrased as 'way TO do X'

**Comment by nschneid** [comment]
>  * no way OF guessing the danger, as a way OF insulting her - I think these could be more easily paraphrased as 'way TO do X'

True, these get a bit into force dynamics/enabling a purpose territory. The "way" is the Means or Manner of achieving a goal. That said it may be easiest to lump this in with Gestalt on the principle that the object is an event whose properties (such as Means) are denoted by the governor. Unless these examples clearly pass one of the tests for Purpose.

**Comment by aryamanarora** [comment]
> new way OF thinking, non-criminal way OF existence - Gestalt

can't you also say "new way TO think", "non-criminal way TO exist" and have it mean the same thing too?
